tests/integration/test_interface.py

```diff
@@ -211,16 +201,11 @@ def test_dataset_filter_by_year(
     """Mock the noising function so that it returns the date column of interest
     with the original (unnoised) values to ensure filtering is happening
     """
-    if "TODO" in dataset_name:
-        pytest.skip(reason=dataset_name)
     year = 2030  # not default 2020
 
-    # Generate a new (non-fixture) dataset for a single year but mocked such
-    # that no noise actually happens (otherwise the years would get noised and
-    # we couldn't tell if the filter was working properly)
-    mocker.patch("pseudopeople.dataset.Dataset._noise_dataset")
```

Comment:
These mocks were added prior to implementing the `config = psp.NO_NOISE` option - but it doesn't work for a distributed cluster since the no_noise call is split among different processes